tests/foreman/installer/test_installer.py

```diff
@@ -1316,14 +1325,27 @@ def test_positive_foreman_module():
     assert rpm_version.replace('-', '.') == semodule_version
 
 
+@pytest.mark.skip_if_open('BZ:1987288')
```

Comment:
I think this BZ should be added to the testimony token in the metadata, since it provides coverage.